app/services/food_service.rb

```diff
@@ -0,0 +1,19 @@
+class FoodService
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |faraday|
+      faraday.params['api_key'] =  ENV['usda_api_key']
+    end
+  end
+
+  def self.total_results(keyword)
+    response = conn.get("foods/search?query=#{keyword}&dataType=Branded&pageSize=10&")
+    output = JSON.parse(response.body, symbolize_names: :true)[:totalHits]
+    # require "pry"; binding.pry
+  end
+
+  def self.top_10_results(keyword)
+    response = conn.get("foods/search?query=#{keyword}&dataType=Branded&pageSize=10&")
+    output = JSON.parse(response.body, symbolize_names: :true)[:foods]
+    # require "pry"; binding.pry
+  end
```

Comment:
These methods are calling to the same endpoint. So, let there just be one method that calls to the endpoint you need, and let your facade pull out the foods and total hits. That way, your service is only responsible for calling to an api endpoint, and your facade is responsible for managing that data.